app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/EateryCard.kt

```diff
@@ -203,11 +211,13 @@ fun EateryCard(
     modifier: Modifier = Modifier
         .fillMaxWidth(),
     isCompact: Boolean = false,
+    isGridView: Boolean = false,
```

Comment:
Really we should probably extract this to an `EateryCardStyle` enum class, so we don't have to do any weird boolean logic.

Reply:
Just addressed all the comments!